scripts/ingests/2020Best_spectraltype_ingest.py

```diff
@@ -0,0 +1,110 @@
+import logging
+# from astroquery.simbad import conf
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+
+SAVE_DB = False  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+
+logger.setLevel(logging.DEBUG)
+
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+# Read in CSV file as Astropy table
+# file = 'Manja19_spectra9.csv'
+file = 'UltracoolSheet-Main.csv'
+data = Table.read('scripts/ingests/' + file)
+# Ingest Optical Spectral Types First
+# use the hashtag to comment it out
+
+# Optical Spectral Types
+filter_opt_data = data[data['spt_opt'] != 'null']
+opt_names = filter_opt_data['name']
+n_sources_opt = len(opt_names)
+opt_regime = ['optical'] * n_sources_opt
+spt_refs_opt = filter_opt_data['ref_spt_opt']
+spectral_types_opt = filter_opt_data['spt_opt']
+# IR Spectral Types
+filter_ir_data = data[data['spt_ir'] != 'null']
+ir_names = filter_ir_data['name']
+n_sources_ir = len(ir_names)
+ir_regime = ['nir_UCD'] * n_sources_ir
```

Comment:
Hm, something is wrong here. it's coming up as `nir_UCD` and I think we just want `nir` but I don't remember exactly.

Reply:
Related to #271 but something we can do cleanup in a separate PR